### ISSUE TITLE
In browse mode, NVDA is not announcing position of menu items (Add domain, Buy domain..etc)

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14179]
### Steps to reproduce:
 1. Start NVDA
 2. Open the above URL in Edge browser.
 3. In focus mode (enable toggle using 'insert+ space'), navigate to "Add domain, Buy domain. Etc"
 4. Observe the behavior of screen reader when focus is on "Add domain, Buy domain..etc".

### Actual behavior:
Command Bar menu bar
 menu item Add domain 
menu item Buy domain
 menu item collapsed Sharing 
menu item Refresh.
In focus mode, NVDA does not announcing position of menu items (Add domain, Buy domain..etc)

### Expected behavior:
In focus mode, NVDA should announce position of menu items (Add domain, Buy domain..etc)
Ex: It should announce as "Add domain menu item 1 of 6"

### System configuration
Code pen: https://codepen.io/testaccount01/pen/jOxLRrz
OS build Version: 21H2 (22000.978)
Edge: Version 105.0.1343.42 (Official build) (64-bit)
Screen reader: NVDA

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2022.2.3

#### Windows version:
Win 11

#### Name and version of other software in use when reproducing the issue:
N/A

#### Other information about your system:
N/A

### Other questions
N/A

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by VivianSi233** [comment]
Similar bug is repro on tablist and filter on https://scc-storybook.azurewebsites.net/?path=%2Fstory%2Ftemplates-workbenchpage--default&viewid=listPage.

![image](https://user-images.githubusercontent.com/89379019/200271446-2ef4384e-20c9-40db-8b23-1b752948b2c9.png)

**Comment by Adriani90** [comment]
This is reproducible in every browser. In general, NvDA seems not to render the number of elements in browse mode at all. in many cases. List items are for example also not counted in browse mode.

**Comment by Adriani90** [comment]
If not all elements are in the DOM, you could try using aria setsize atribute to overwrite the item count which is automatically calculated by the browser itself. But in focus mode this codepen seems to work properly, so Aria setsize is not needed in this case because all elements are in the DOM. And even using aria setsize atribute, NVDA will very probably announce the item count only in focus mode, not in browse mode.
See this link for more details:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-setsize

@jcsteh just out of curiosity, what was the reason for not considering item position in browse mode when this was implemented?

**Comment by Adriani90** [comment]
If this gets implemented, I suggest to announce the item position (x of y) only when using quick navigation key. You can assign a quick navigation key for menu in input gestures dialog, but there is no quick navigation key for menu items yet.
